skt/runner.py

```diff
@@ -167,6 +180,15 @@ def _forget_cid(self, cid):
             raise ValueError("Unknown cid type: %s" % cid)
 
     def getverboseresults(self, joblist):
+        """
+        Retrieve verbose results for a list of beaker jobs
+
+        Args:
+            joblist: A list of beaker jobs
```

Comment:
I think this needs to be described more specifically and using official terms. E.g. "joblist: A list of Beaker job taskspecs (universal IDs)." This will have to do for now, though.